### DISCUSSION TITLE
Unexpected Transcriptions from Deepgram Tamil STT with Pipecat Integration

### DISCUSSION THREAD

**Discussion by developerKylient** [deepgram/community#1026] [General help]
I'm encountering an issue while using Deepgram's Tamil STT model integrated with Pipecat. Without any actual speech input, the system continuously transcribes phrases like "புதியதோர் உலகம்" (translated as "The New World") in the logs. This happens even when there's complete silence or minimal background noise.

**Comment by deepgram-community** [comment]
Thanks for asking your question. Please be sure to reply with as much detail as possible so the community can assist you efficiently.
<sub>Consider joining our [Discord community](https://discord.gg/deepgram) for more opportunity to engage with your fellow Deepgram users. You can earn points which can be redeemed for cool stuff by being active in our communities!</sub>

**Comment by deepgram-community** [comment]
Hey there! It looks like you haven't connected your GitHub account to your Deepgram account. You can do this at https://community.deepgram.com - being verified through this process will allow our team to help you in a much more streamlined fashion.

**Comment by deepgram-community** [comment]
It looks like we're missing some important information to help debug your issue. Would you mind providing us with the following details in a reply?

- The programming language you are working in (e.g. JavaScript, Python).
- A request ID that triggered your error or issue.

**Comment by deepgram-community** [comment]
could you provide some request IDs  so we can take a deeper look?

> ![avatar](https://avatars.slack-edge.com/2023-02-20/4855338233600_fad9174023ee3f640ce1_32.png) This message was sent by John Vajda from Deepgram, via our community automation.
